src/Metrics.ts

```diff
@@ -0,0 +1,109 @@
+import {Gauge} from "prom-client";
+
+export class Metrics {
+     githubRepoGauge: Gauge;
+     githubRepoPublicGauge: Gauge;
+     githubRepoPrivateGauge: Gauge;
+     githubRepoStarsGauge: Gauge;
+     githubRepoForksGauge: Gauge;
+     githubRepoOpenIssuesGauge: Gauge;
+     githubRepoPullRequestsGauge: Gauge;
+    githubRepoPullRequestsOpenGauge: Gauge;
+     githubRepoPullRequestsCloseGauge: Gauge;
+     githubPullRequestsGauge: Gauge;
+     githubPullRequestsOpenGauge: Gauge;
+     githubPullRequestsCloseGauge: Gauge;
+     githubPullRequestsOpenApprovedGauge: Gauge;
+     githubPullRequestsOpenWaitingApprovalGauge: Gauge;
+     githubPullRequestsMergedGauge: Gauge;
+     githubRepoBranchesGauge: Gauge;
```

Comment:
Je sais je gosse avec les readonly, mais ça prévient les accidents et ça limite le scope de l'"API" public upfront. C'est une app privé donc whatever. Par contre si faut prendre l'habitude sinon ça s'oublie facilement quand c'est plus important.

Aussi j'enlèverais tous les préfixes "github". C'est assez obvious dans un "exporter-github".

```suggestion
    readonly repoGauge: Gauge;
    readonly repoPublicGauge: Gauge;
    readonly repoPrivateGauge: Gauge;
    readonly repoStarsGauge: Gauge;
    readonly repoForksGauge: Gauge;
    readonly repoOpenIssuesGauge: Gauge;
    readonly repoPullRequestsGauge: Gauge;
    readonly repoPullRequestsOpenGauge: Gauge;
    readonly repoPullRequestsCloseGauge: Gauge;
    readonly pullRequestsGauge: Gauge;
    readonly pullRequestsOpenGauge: Gauge;
    readonly pullRequestsCloseGauge: Gauge;
    readonly pullRequestsOpenApprovedGauge: Gauge;
    readonly pullRequestsOpenWaitingApprovalGauge: Gauge;
    readonly pullRequestsMergedGauge: Gauge;
    readonly repoBranchesGauge: Gauge;
```